src/docs/development/ui/navigation.md

```diff
@@ -6,14 +6,23 @@ title: Navigation & routing
 TODO: once Cookbook page navigation is in place, change the following to be an autogenerated list.
 {% endcomment -%}
 
-For a discussion of how to use Flutter's Navigator class with various types of routes, see the [API documentation on the Navigator class]({{site.api}}/flutter/widgets/Navigator-class.html).
+For a discussion of how to use Flutter's Navigator class with
+various types of routes, see the
+[API documentation on the Navigator class][].
 
 The following [cookbook recipes cover {{page.title | downcase }}][cookbook]:
```

Comment:
Hmmm... it's weird to see "&" in text. Maybe the page name should be "Navigation and routing"?

Usually I avoid "&" in titles & text except in the side nav. (oh hey, that's official: https://developers.google.com/style/word-list#letter-symbol)